Entity/NotificationMessageInterface.php

```diff
@@ -1,11 +1,10 @@
 <?php
 
-namespace Kopay\NotificationBundle\Entity;
+  namespace Kopay\NotificationBundle\Entity;
 
-use Doctrine\Common\Collections\Collection;
+  use Doctrine\Common\Collections\Collection;
 
-interface NotificationMessageInterface
-{
+  interface NotificationMessageInterface {
```

Comment:
brace to the new line please. Let's follow PSR-2.

Reply:
ok, agree